netmanager/src/views/pages/CohortsRegistry/CohortDetails.jsx

```diff
@@ -186,7 +186,7 @@ const CohortForm = ({ cohort }) => {
   const initialState = {
     name: '',
     network: activeNetwork.net_name,
-    visibility: false
+    visibility: null
```

Comment:
Hi @Codebmk , thanks for this.

- Is null the most appropriate default value for a Boolean field?
- Could you share some more insights behind this thought process of this hotfix?

Reply:
The default was initially false but that is not the case with the backend. If there's no visibility value, it is empty. Hence using null in the frontend. @Baalmart

Reply:
Okay @Codebmk . thanks for the clarifications. As long as things work as expected. Let us continue to study the situation.